content/en/docs/chart_template_guide/functions_and_pipelines.md

```diff
@@ -190,6 +190,72 @@ logic that is a little more sophisticated than just inserting a string. In the
 next section we will look at the control structures provided by the template
 language.
 
+## Using the `lookup` function
+
+The `lookup` function can be used to resources from resources in a running cluster. The synopsis of
```

Comment:
> The `lookup` function can be used to resources from resources in a running cluster.

Doesn't make sense. I think it means to say...

> The `lookup` function can be used to _lookup_ resources in a running cluster.

Though, some other variation may be what was meant or better.